README.md

```diff
@@ -0,0 +1,8 @@
+# Air Console
+
+A Airbnb clone made while attending the LeWagon 9 week Fullstack program.
```

Comment:
You should fix your grammar.. Change 'A' to 'An'

Reply:
Okay, will change!